docs/ci.md

```diff
@@ -16,3 +16,18 @@
 - Do integration and acceptance tests on PR requests
 - Deploy on a merge to master to an environment, (whether ephemeral or static) to perform a light end-to-end, not repeating the above steps
 - Perform quick end-to-end tests that are there to just validate that things work together as expected, (no need to repeat unit / acceptance tests already done)
+- Have clear, concise commit messages conforming to [semantic commit messages](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#), including a JIRA ID and/or username of the other member of the pair committing the change (`git commit --verbose' is useful for multi-line messages)
```

Comment:
I don't think we want to mention JIRA ID specifically here as we want to try and be agnostic of any tooling in our principles; so they are applicable to any tool.